package.json

```diff
@@ -51,6 +51,7 @@
     "bcrypt": "5.0.1",
     "bluebird": "3.7.2",
     "burner-email-providers": "1.0.67",
+    "chai-things": "^0.2.0",
```

Comment:
Why is this dependency needed?

Reply:
I used it to test for the propery isActive on an array of projects and events

Reply:
Doesn't seem needed as a main dependency in the project if it's only used for tests.

Reply:
moved to devDependencies